.github/workflows/golangci-lint.yml

```diff
@@ -23,9 +23,8 @@ jobs:
       - uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # v5.0.2
         with:
           go-version: ${{ steps.goversion.outputs.goversion }}
-      - uses: actions/checkout@v4
```

Comment:
Note that this step is duplicated, as it is already defined on line 20.

https://github.com/etcd-io/gofail/blob/69df3ad24cabd750c11358af14381eb454afecd8/.github/workflows/golangci-lint.yml#L20-L26